Tasks/DownloadGitHubNpmPackageV1/task.json

```diff
@@ -1,16 +1,19 @@
 {
     "id": "97411e3d-0241-4b1f-9607-2d2c04b4df51",
     "name": "DownloadGithubNpmPackage",
-    "friendlyName": "Download Github Npm Package",
-    "description": "Install npm packages from GitHub.",
+    "friendlyName": "Download Github Npm Packages",
+    "description": "Install npm packages from GitHub. This task doesn't support package download from other npm registries.",
     "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/package/npm",
     "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=613746) or [see the npm documentation](https://docs.npmjs.com/)",
     "category": "Package",
     "author": "Microsoft Corporation",
+    "visibility": [
+        "Build"
+    ],
     "version": {
         "Major": 1,
         "Minor": 0,
```

Comment:
You may want to follow Sprintly model for versioning. Its easier to maintain specially with hot fixes.

Minor version - Sprint in which the task got updated
Patch version - Patch version of that Sprint, starts with 0.